src/components/editor/property/PropertyTemplateOutline.jsx

```diff
@@ -51,13 +51,17 @@ export class PropertyTemplateOutline extends Component {
   // When the plus button is clicked, load reference templates (property.valueConstraint.valueTemplateRefs)
   handleClick = property => (event) => {
     event.preventDefault()
-    this.setState({ collapsed: !this.state.collapsed })
 
-    const templateRefList = isResourceWithValueTemplateRef(property) ? property.valueConstraint.valueTemplateRefs : []
+    if (this.state.collapsed && !this.state.added) {
+      const templateRefList = isResourceWithValueTemplateRef(property) ? property.valueConstraint.valueTemplateRefs : []
 
-    this.fulfillRTPromises(this.resourceTemplatePromises(templateRefList)).then(() => {
-      this.addPropertyTypeRows(this.props.propertyTemplate)
-    })
+      this.fulfillRTPromises(this.resourceTemplatePromises(templateRefList)).then(() => {
+        this.addPropertyTypeRows(this.props.propertyTemplate)
+      })
+      this.setState({ added: true })
```

Comment:
Is it possible to look at the redux store to tell if this is added without having to keep a separate piece of state around?

Reply:
I'll take a look. My first attempt at that led to a different bug, I'll time box working through that and report back soon.

Reply:
@jcoyne using the existing `rowAdded` state worked instead of creating a new state value. How do you feel about that?

Reply:
@aaron-collier and @jcoyne is this now resolved or still open?

Reply:
Yes, no problem with this.